<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManager2Class</class>
 <widget class="QWidget" name="TaskManager2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManager2</string>
  </property>
  <widget class="QListWidget" name="list_tasks">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tasks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_description">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_id">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Programmer's id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_done">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TaskManager2.qrc"/>
 </resources>
 <connections/>
</ui>
